src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar--start-day--sat calendar--month-length--">
```

Comment:
```suggestion
      <div class="calendar calendar--start-day--sat calendar--month-length--31">
```
Let's add some value initially